### DISCUSSION TITLE
Bet365 is blocking (again)

### DISCUSSION THREAD

**Discussion by javinair** [kaliiiiiiiiii/Selenium-Driverless#173] [General]
Hi there,

I've been developing normally yesterday and today. But from a few minutes ago, I get the following advice:

![image](https://github.com/kaliiiiiiiiii/Selenium-Driverless/assets/44604000/80e082fb-0048-4a54-a64b-1bac5a67ef18)

I'm currently using the AsyncIO version.
Any workaround/solution?

Thanks.

**Comment by ttlns** [comment]
I'll give it a test in some time. If you don't hear anything from me, feel free to remind me again:)

**Comment by ttlns** [comment]
@javinair Did you check if your IP is being flagged?

**Reply by javinair** [reply]
Hello @kaliiiiiiiiii ,

As you thought, it was because of a "soft" IP blocking. I say soft because after this problem, I tried with my PC and I had the same issue. But I could login as usual using my phone and the official app with the same WiFi. After this, I could play again with your selenium without any problem.

So... false alarm, sorry.

**Reply by ttlns** [reply]
great, no worries then:)
